### PR TITLE
Stop updating slugs on existing pages.

### LINKED ISSUE #528: Slug change no warning
When you change the name of a page, it would be nice if you get a warning message telling the user  that the slug will change, or even better, get a choice if you want it to change or not.

### PR DESCRIPTION
**Pull request by BertrandBordage**
Fixes #528.

The only exception is when the user deliberately cleared the field or when something wrong happened in a script, causing pages to have empty slugs.

### CODE REVIEW

**Comment by gasman** [comment]
Having had a play around with this, I think the behaviour isn't quite there yet...

* On page creation, editing the title always causes the slug to be overwritten, even if you've edited the slug manually
* I think the fields should continue to be linked after the page is saved, until the page is published (or the user explicitly changes the slug). That way, if the editor saves a draft and then fixes a spelling mistake in the title, the slug will get fixed too.

In other words, I think the preferred behaviour would be: keep the auto-updating in force, unless

* the page is currently live (ideally this should be "the page has ever been published", but I don't think we have any way to determine that at the moment) - OR
* the slug has been edited so that it doesn't match the title.

**Comment by chrxr** [comment]
Hi @BertrandBordage, please could you make the changes @gasman suggests and resubmit.

**Comment by BertrandBordage** [comment]
Hi @chrxr: of course, but I’m working on another Torchbox project for the moment. You have to see with Leon if that’s ok, I guess.

**Comment by gasman** [comment]
Closing in favour of #1810
